tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionTranspilerSuite.scala

```diff
@@ -489,10 +509,18 @@ class RegularExpressionTranspilerSuite extends FunSuite with Arm {
       val cpu = cpuReplace(javaPattern, input)
       val cudfPattern = new CudfRegexTranspiler(RegexReplaceMode).transpile(javaPattern)
       val gpu = try {
-        gpuReplace(cudfPattern, input)
+        val result = Future {
+          gpuReplace(cudfPattern, input)
+        }
+        Await.result(result, 10 second)
```

Comment:
Is there a reason why the earlier test has a different timeout from this test? Are these timeouts lenient enough to cope with temporary slowdowns (perhaps due to garbage collection pauses) when these tests are running in CI?